vite.config.ts

```diff
@@ -1,10 +1,22 @@
 import { defineConfig } from "vitest/config";
 import { sveltekit } from "@sveltejs/kit/vite";
+import { svelteTesting } from "@testing-library/svelte/vite";
 
 export default defineConfig({
-    plugins: [sveltekit()],
-
+    plugins: [sveltekit(), svelteTesting()],
+    // Tell Vitest to use the `browser` entry points in `package.json` files, even though it's running in Node
+    resolve: process.env.VITEST ? { conditions: ["browser"] } : undefined,
     test: {
-        include: ["test/**/*.{test,spec}.{js,ts}"],
+        reporters: process.env.GITHUB_ACTIONS ? ["default", "github-actions"] : ["default"],
+        sequence: {
+            shuffle: true,
+        },
+        coverage: {
+            include: ["src/**"],
```

Comment:
Why now include src/** and not test/**?

Reply:
Here we define which files are tracked for the coverage report. Without this, some config files in the root directory will be included.

Reply:
Ah ok, makes sense